core/schema/service.graphqls

```diff
@@ -44,7 +44,10 @@ type Service {
   """
   Stop the service.
   """
-  stop: ServiceID!
+  stop(
+    "Immediately kill the service without waiting for a graceful exit"
+    kill: Boolean
+  ): ServiceID!
```

Comment:
Bikeshedding welcome :tada: This is the "easiest" possible interface - users can implement their own graceful stop on top of this if they like.

However, it could also be nice to have a default graceful stop similarly to this PR implements internally (but doesn't expose). One problem with this though is it means we'd really want to pass `time.Duration`s through the API (I think this is the first case?) - ideally each SDK would choose how to work with datetime, instead of passing around magic strings/integers. This feels like a big can of worms that I don't really want to open right now.